manifests/latest/vsphere-7.0u1/vanilla/deploy/vsphere-csi-node-ds.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       dnsPolicy: "Default"
       containers:
       - name: node-driver-registrar
-        image: quay.io/k8scsi/csi-node-driver-registrar:v1.3.0
+        image: quay.io/k8scsi/csi-node-driver-registrar:v2.1.0
         lifecycle:
           preStop:
             exec:
```

Comment:
In https://github.com/kubernetes-csi/node-driver-registrar/blob/release-2.0/CHANGELOG/CHANGELOG-2.0.md we can see this doc - Registration socket is removed on node-driver-registrar shutdown, de-registering the CSI driver from kubelet. Looks like we dont need the preStop.

Reply:
Yes, it is not needed. let me remove prestop as per https://github.com/kubernetes-csi/node-driver-registrar/pull/61/files